.gitpod.yml

```
@@ -0,0 +1,4 @@
+tasks:
+  - init: npm install && npm run build
+image:
+  file: .gitpod.Dockerfile
```

@PappuKP The setup can be improved as shown below. 
```suggestion
tasks:
  - init: npm install
    command: npm run develop
ports: 
  - port: 8000
    onOpen: open-preview
image:
  file: .gitpod.Dockerfile
```